apps/mosaico/src/Mosaico/Footer.purs

```diff
@@ -258,13 +258,18 @@ column children = DOM.div { className: "mosaico-footer__contact-info__column", c
 
 section :: String -> Array JSX -> JSX
 section title children =
-  DOM.p
```

Comment:
Here's a real explanation of what's going on: https://stackoverflow.com/questions/20134929/div-tag-inside-of-p-looking-weird-in-the-dom-tree

It's not valid HTML to have P contain a DIV. It's not a react but a HTML thing.